dallinger/command_line.py

```diff
@@ -169,9 +188,10 @@ def setup_experiment(debug=True, verbose=False, app=None):
         os.path.join(dst, "dallinger_experiment.py"))
 
     # Copy files into this experiment package.
-    src = os.path.join(
-        os.path.dirname(os.path.realpath(__file__)),
-        "custom.py")
+    from pkg_resources import get_distribution
+    dist = get_distribution('dallinger')
+    src_base = os.path.join(dist.location, dist.project_name)
+    src = os.path.join(src_base, "custom.py")
     shutil.copy(src, os.path.join(dst, "custom.py"))
```

Comment:
FYI, once we merge in story 142 this goes away, because the routes are loaded as part of the main dallinger flask app in `experiment_server`, rather than as part of a custom blueprint.